examples/nlp/language_modeling/megatron_gpt_ptune.py

```diff
@@ -1,135 +0,0 @@
-# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
-#
```

Comment:
Since `megatron_gpt_ptune.py` is removed completely. Could you make sure the notebook `https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/PTune_multiple_NLP_tasks.ipynb` is still working? It needs to be updated.

Reply:
Yes, I am planning to do this in a separate PR though because this one is already pretty large. I am working on a new notebook showing how to use both methods. Do you think we should still maintain the ptuning notebook and have two example notebooks for people to use, or would that be confusing for new users?